README.md

```
@@ -46,6 +46,13 @@
 1. Copy the `hyxi_cloud` folder to your `/config/custom_components/` directory.
 2. Restart Home Assistant.
 
+### Trying Upcoming Features (Beta Channel)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)